### ISSUE TITLE
Is `settings.SongHint` really necessary?

### ISSUE THREAD

**Issue by ccuser44** [Epix-Incorporated/Adonis#1744]
### What part of Adonis is this related to?

Loader/Settings

### What are you suggesting?

Do we really need it? Maybe? Maybe not?

**Comment by GalacticInspired** [comment]
Yes we do, it is a setting for a reason. Some experiences, like theater games, may want to notify all users when a host is playing music, while some games may not want to do that.
